docs/technical_reference/unit_models/electrodialysis_0D.rst

```diff
@@ -0,0 +1,219 @@
+
+   "Temperature, inlet_diluate", ":math:`T`", "temperature", "None", ":math:`K`", 1
+   "Temperature, inlet_concentrate", ":math:`T`", "temperature", "None", ":math:`K`", 1
+   "Pressure, inlet_diluate",":math:`p`", "temperature", "None", ":math:`Pa`", 1
+   "Pressure, inlet_concentrate",":math:`p`", "temperature", "None", ":math:`Pa`", 1
+   "Component molar flow rate, inlet_diluate", ":math:`N`", "flow_mol_phase_comp", "[[t], ['Liq'],['H2O', 'Na_+', '\Cl_-']", ":math:`mol s^{-1}`", 3
+   "Component molar flow rate, inlet_concentrate", ":math:`N`", "flow_mol_phase_comp", "[[t], ['Liq'],['H2O', 'Na_+', '\Cl_-']", ":math:`mol s^{-1}`", 3
+   "Water transport number", ":math:`t_w`", "water_trans_number_membrane", "['cem', 'aem']", "dimensionless", 2
+   "Water permeability", ":math:`L`", "water_permeability_membrane", "['cem', 'aem']", ":math:`m^{-1}s^{-1}Pa^{-1}`", 2
+   "Voltage or Current \ :sup:`2`", ":math:`U` or :math:`A`", "voltage or current", "[t]", ":math:`\text{V}` or :math:`A`", 1
+   "Electrode areal resistance", ":math:`r_{el}`", "electrodes_resistance", "[t]", ":math:`\Omega m^2`", 1
+   "Cell pair number", ":math:`n`", "cell_pair_num", "None", "dimensionless", 1
+   "Current utilization coefficient", ":math:`\xi`", "current_utilization", "None", "dimensionless", 1
+   "Spacer thickness", ":math:`s`", "spacer_thickness", "none", ":math:`m` ", 1
+   "Membrane areal resistance", ":math:`r`", "membrane_surface_resistance", "['cem', 'aem']", ":math:`\Omega m^2`", 2
+   "Cell width", ":math:`b`", "cell_width", "None", ":math:`\text{m}`", 1
+   "Cell length", ":math:`l`", "cell_length", "None", ":math:`\text{m}`", 1
+   "Thickness of ion exchange membranes", ":math:`\delta`", "membrane_thickness", "['cem', 'aem']", ":math:`m`", 2
+   "diffusivity of solute in the membrane phase", ":math:`D`", "solute_diffusivity_membrane", "[['cem', 'aem'], ['Na_+', '\Cl_-']]", ":math:`m^2 s^{-1}`", 4
+   "transport number of ions in the membrane phase", ":math:`t_i`", "ion_trans_number_membrane", "[['cem', 'aem'], ['Na_+', '\Cl_-']]", "dimensionless", 4
+
+**Note**
+ :sup:`1` DOF number takes account of the indices of the corresponding parameter.
+ :sup:`2` A user should provide either current or voltage as the electrical input, in correspondence to the "Constant Current"
+ or "Constant Voltage" treatment mode (configured in this model). The user also should provide an electrical magnitude
+ that ensures a operational current *below the limiting current* of the feed solution.
+
+
+Solution component information
+------------------------------
+To fully construct solution properties, users need to provide basic component information of the feed solution to use
+this model, including identity of all solute species (i.e., Na :sup:`+`, and \Cl :sup:`-` for a
+NaCl solution), molecular weight of all component species (i.e., H\ :sub:`2`\ O, Na :sup:`+`, and \Cl :sup:`-`), and charge
```

Comment:
Ideally, we should link to the documentation for the ion property package (since this dict is used on the property package and not the unit model), however, there is no documentation for the ion property package at the moment.

I think it is fine to leave this note in here for now, but we need to open an issue about the ion property package documentation. Once that documentation is in place, we should just link to that documentation here.

Reply:
Good idea and I will revisit this when there is a documentation for the property pack.